charts/kubermatic/static/master/versions.yaml

```diff
@@ -5,17 +5,25 @@ versions:
   version: 1.16.13
 - type: kubernetes
   version: 1.16.14
+- type: kubernetes
+  version: 1.16.15
 - type: kubernetes
   version: 1.17.9
 - type: kubernetes
   version: 1.17.11
+- type: kubernetes
+  version: 1.17.12
 - type: kubernetes
   version: 1.18.6
+- type: kubernetes
+  version: 1.18.8
 - default: true
   type: kubernetes
-  version: 1.18.8
+  version: 1.18.9
 - type: kubernetes
   version: 1.19.0
+- type: kubernetes
+  version: 1.19.2
```

Comment:
As far as I see, we try to support as many patch releases as possible. To keep this consistent, should we also add support for 1.19.1? I don't see any value in doing that, so this is only for the sake of consistency.

Reply:
I'm not sure if/why anyone would want to use 1.19.1 if 1.19.2 is supported? But i don't know ^^

Reply:
That's what I was thinking as well and that's why I just said for the sake of consistency. We can leave it as it is right now.

Reply:
One argument against it: I cannot easily have one PR that tests both patch releases ;-)